proto/lib/python/none.proto

```diff
@@ -5,5 +5,5 @@ import "proto/core/common/common_object.proto";
 
 
 message None {
-    syft.core.common.UID id_at_location = 2;
+    syft.core.common.UID id = 2;
```

Comment:
wow good call - not sure when this got changed

Reply:
Q: Can't we use "1" here?